R/detect_structure.R

```diff
@@ -29,8 +29,12 @@
 #' detected_structure
 detect_structure <- function(reporting_triangle) {
   n_row_nas <- sum(is.na(rowSums(reporting_triangle)))
-  # Structure is NULL if there are no NAs
+  # Structure is 0 if there are no NAs
   if (n_row_nas == 0) {
+    cli_warn(
```

Comment:
At most this seems like it should be a message. I would advocate for nothing here due to the many other use cases for this data restructure.